enos/enos-variables.hcl

```diff
@@ -91,8 +91,11 @@ variable "tfc_api_token" {
 }
 
 variable "vault_artifact_type" {
-  description = "The Vault artifact type package or bundle"
+  description = "The Vault artifact type"
   default     = "bundle"
+  validation {
```

Comment:
validations don't work for enos variables. I'm actually quite surprised this doesn't result in an error because there's a child block present in the variable block.

Reply:
Ooh, didn't know that. I'll remove it.